Gemfile.lock

```diff
@@ -0,0 +1,146 @@
+PATH
```

Comment:
Do we really want to include the `Gemfile.lock` in the repo?  For a gem like this one that looks odd to me…

Reply:
Odd. I thought I had excluded it. Will remove.